tripal_chado/includes/tripal_chado.field_storage.inc

```diff
@@ -1139,6 +1156,38 @@ function tripal_chado_field_storage_bundle_mapping_form_summary(&$form, &$form_s
     ];
   }
 
+  //
+  // Add the row for cvterm options
+  //
+  if ($default['use_cvterm']) {
+    $message = 'No. All records belong to a single controlled vocabulary.';
+    if ($default['use_cvterm'] == 'parent') {
+      $message = 'Yes. Records should include all child records of the specified term.';
```

Comment:
I think I see what direction this is headed in, but does it really make sense to include?  Is the use case something like having a "branch" of a CV dedicated to user-submitted terms?  Is that really the right solution as opposed to a user-specific CV (which would be the other case)?

If theres not a good use for this alternate schema i propose dropping.

Reply:
I'm good with dropping this option as I don't actually need it... @spficklin since you originally added it, do you think it still needs to stay?

Reply:
Sorry, I don't understand the question....

Reply:
The highlighted code is just printing summary info, so I'm not making the connection with the question.

Reply:
We are discussing the "All child records of the specified term" option.

![screen shot 2019-01-04 at 6 42 07 pm](https://user-images.githubusercontent.com/7063154/50716638-7d78af00-1050-11e9-8548-aba00d1df812.png)

Reply:
Oh, I see. Yes, we can get rid of it. It doesn't work so no point in offering the option when it doesn't work.